dtde/spark/cli/cluster_create.py

```diff
@@ -30,29 +30,52 @@ def setup_parser(parser: argparse.ArgumentParser):
 
 
 def execute(args: typing.NamedTuple):
+    # read cluster.yaml configuartion file, overwrite values with args
+    confg_obj = ConfigObject()
+    confg_obj.read_config_file()
+
+    if args.cluster_id is not None:
```

Comment:
hhm, here I would have something more in the lines of config_obj.merge(
size=args.size,
...
)